### PR TITLE
update various unmaintained Python ports

### PR DESCRIPTION
**Pull request by reneeotten**
#### Description
- see commit messages for a description of the updates

###### Tested on

macOS 10.14.3 18D42
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? N/A
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`? Yes, where applicable
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-novaclient/Portfile [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-                     port:py${python.version}-httplib2 \
-                     port:py${python.version}-simplejson \
-                     port:py${python.version}-iso8601 \
-                     port:py${python.version}-setuptools
+    depends_lib-append \
+                    port:py${python.version}-prettytable \
+                    port:py${python.version}-httplib2 \
+                    port:py${python.version}-simplejson \
+                    port:py${python.version}-iso8601 \
+                    port:py${python.version}-setuptools
```
Comment (mf2k) [inline python/py-novaclient/Portfile]: Since you are in here, please sort these dependencies alphabetically.

**Comment by reneeotten** [comment]
```py-kcs11``` failure... it likely needs ```swig-python``` instead of ```swig``` only

File: python/py-novaclient/Portfile (hunk 1 above)
Comment (reneeotten) [reply python/py-novaclient/Portfile]: okay, done

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**